src/space.js

```diff
@@ -122,6 +122,19 @@ const publicStoreReducer = (store) => {
       throwIfUndefined(key, 'key')
       store.set(PREFIX + key, value)
     },
+    setMultiple: async (keys, values) => {
+      throwIfNotEqualLenArrays(keys, values)
+      try {
+        await keys.reduce(async (previousPromise, nextKey, index) => {
+          await previousPromise
+          throwIfUndefined(nextKey, 'key')
+          return store.set(PREFIX + nextKey, values[index])
+        }, Promise.resolve())
+        return true
```

Comment:
Why not do `keys = keys.map(e => PREFIX + e)` then call `store.setMultiple`?

Reply:
Made this change, but what happens if key is a number? Is it converted to a string somewhere?

Reply:
👍 Here it will be converted to a string since we `+` with a string. However it's a good question if a number is passed into orbitdb as a key. I'm not completely sure about that.